web/html/src/manager/content-management/shared/business/filters.enum.js

```diff
@@ -42,7 +48,8 @@ function findByKey(key: string): FilterOptionType {
 function getFilterDescription (filter: Object): string {
   const matcherDescription = {
     equals: " matching ",
-    contains: " containing "
+    contains: " containing ",
+    greater: " greater "
```

Comment:
I think this should be adjusted with the latest commit too (both the key and the description).

Reply:
yes, need to be alligned otherwise the filter description in a project is wrong. Fixed